src/config/schema.rs

```diff
@@ -1,17 +1,22 @@
-use serde::{Deserialize, Serialize};
+use vector_config::configurable_component;
 use vector_core::config::LogNamespace;
 
 pub(crate) use crate::schema::Definition;
 
-#[derive(Debug, Deserialize, Serialize, PartialEq, Eq, Copy, Clone)]
+/// Schema options.
```

Comment:
😵‍💫 just realizing how this naming collides with the configurable schema

Reply:
Hmmm, how so?

Reply:
Just that I thought this file was about generating Vector's schema on first glance, not enforcing an event schema. Perhaps due to how much I've been looking at the JSONSchema work.

Reply:
Ahhhh, gotcha. 👍🏻